jsonnet/kube-prometheus/kube-prometheus-eks.libsonnet

```diff
@@ -1,63 +1,70 @@
-local k = import 'github.com/ksonnet/ksonnet-lib/ksonnet.beta.4/k.libsonnet';
-local service = k.core.v1.service;
-local servicePort = k.core.v1.service.mixin.spec.portsType;
-
 {
   _config+:: {
     eks: {
       minimumAvailableIPs: 10,
-      minimumAvailableIPsTime: '10m'
-    }
+      minimumAvailableIPsTime: '10m',
+    },
   },
   prometheus+: {
     serviceMonitorCoreDNS+: {
-        spec+: {
-          endpoints: [
-            {
-              bearerTokenFile: "/var/run/secrets/kubernetes.io/serviceaccount/token",
-              interval: "15s",
-              targetPort: 9153
-            }
-          ]
+      spec+: {
+        endpoints: [
+          {
+            bearerTokenFile: '/var/run/secrets/kubernetes.io/serviceaccount/token',
+            interval: '15s',
+            targetPort: 9153,
+          },
+        ],
+      },
+    },
+    AwsEksCniMetricService: {
+      apiVersion: 'v1',
+      kind: 'Service',
+      metadata: {
+        name: 'aws-node',
+        namespace: 'kube-system',
+        labels: { 'k8s-app': 'aws-node' },
+      },
+      spec: {
+        ports: [
+          { name: 'cni-metrics-port', port: 61678, targetPort: 61678 },
+        ],
+        selector: { 'k8s-app': 'aws-node' },
+        clusterIP: 'None',
+      },
+    },
+
+    serviceMonitorAwsEksCNI: {
```

Comment:
nit: we should change the name to `AwsEksCniMetricServiceMonitor` to be consistent with `AwsEksCniMetricService`